public/src/js/config.js

```diff
@@ -69,6 +69,13 @@ angular.module('lisk_explorer')
           $rootScope.title = $route.current.title;
           $rootScope.isCollapsed = true;
 
+          // Market Watcher
+          $http.get('/api/exchanges').then (function (result) {
+              if (result.data.success && result.data.enabled) {
+                $rootScope.marketWatcher = true;
```

Comment:
We need to store that in $rootScope, that variable will define it there is need to display `Market Watcher` position in `Tools` menu.